frontend/src/mock-cloud/fixtures.ts

```
@@ -0,0 +1,57 @@
+// Dev-only fixtures for VITE_MOCK_CLOUD mode. Tree-shaken out of production builds.
+import type {
+  DashboardProjectInfo,
+  InsForgeDashboardProps,
+} from '@insforge/dashboard';
```

ESLint

[error] Replace `⏎··DashboardProjectInfo,⏎··InsForgeDashboardProps,⏎` with `·DashboardProjectInfo,·InsForgeDashboardProps·`

(prettier/prettier)